tests/core/test_infer.py

```diff
@@ -46,11 +48,11 @@ def __init__(self):
 
     @property
     def input_ports(self):
-        return {"mod_in": NeuralType({0: AxisType(BatchTag), 1: AxisType(BaseTag, dim=1)})}
+        return {"mod_in": NeuralType(ChannelType(), (AxisType(AxisKind.Batch), AxisType(AxisKind.Dimension, 1)))}
 
     @property
     def output_ports(self):
-        return {"mod_out": NeuralType({0: AxisType(BatchTag), 1: AxisType(BaseTag, dim=1)})}
+        return {"mod_out": NeuralType(ChannelType(), (AxisType(AxisKind.Batch), AxisType(AxisKind.Dimension, 1)))}
```

Comment:
why is this not using axis=(...) notation?

Reply:
specifying kwarg's argument name is not required, here for brevity, I didn't use it